ArduinoFrontend/src/app/Libs/Workspace.ts

```diff
@@ -555,6 +555,7 @@ export class Workspace {
       x - offsetX,
       y - offsetY
     );
+    console.log(obj);
```

Comment:
delete all console.log statements.

Reply:
My bad, In next time i remove the comment